bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/BookKeeper.java

```diff
@@ -66,10 +65,9 @@ static BookKeeperBuilder newBuilder(final ClientConfiguration clientConfiguratio
     /**
      * Close the client and release every resource.
      *
-     * @throws BKException
      * @throws InterruptedException
      */
     @Override
-    void close() throws BKException, InterruptedException;
```

Comment:
if you already introduce the new exception class in api package, you don't need to drop this signature, no?

Reply:
ok